gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -312,3 +313,19 @@ def _get_merged_field_schemas(
         existing_field_schema.fields = _get_merged_field_schemas(
             existing_field_schema.fields, field_schema.fields)
   return merged_field_schemas
+
+
+def calculate_optimal_partition_size(total_base_pairs):
```

Comment:
Since this is a public facing method, add type def. Plus a doc string with explanation why we are doing these calculations would be preferred.